tools/dbTools.py

```diff
@@ -115,9 +124,21 @@ def is_scraped(self, file_hash):
         )
         result = self.cursor.fetchone()
         if result:
-            return result.id, result.scrape_again
+            self.logger.info('Got a result')
+            self.logger.info(result.scrape_again)
+            self.logger.info(type(result.scrape_again))
+            self.logger.info(dir(result.scrape_again))
+            return {
+                 'id': result.id,
+                 're-scrape': result.scrape_again,
+                 'status': True
+            }
```

Comment:
Are these dicts only passed around internally, or are they serialised externally?

If the latter: prefer passing around custom objects (or even better, using exceptions for control flow) – this opens the door to tools like pylint and mypy for code checking.

Reply:
It is only passed internally